### PR TITLE
[Metricbeat] Add usage metricset into aws module

### PR DESCRIPTION
**Pull request by kaiyan-sheng**
CloudWatch collects metrics that track the usage of some AWS resources. These metrics correspond to AWS service quotas and are collected every minute.

Predefined dashboard is also available for this metricset:
localhost_5601_app_kibana (3)

## How to test this
Enable aws module and change aws.yml to:
```
- module: aws
  period: 1m
  metricsets:
    - usage
  credential_profile_name: test-mb
```
closes https://github.com/elastic/beats/issues/14935

### CODE REVIEW

File: x-pack/metricbeat/module/aws/_meta/kibana/7/dashboard/Metricbeat-aws-usage-overview.json [hunk 1]
```diff
@@ -0,0 +1,823 @@
...
+        "version": 1,
+        "visState": {
+          "aggs": [],
+          "params": {
+            "controls": [
+              {
+                "fieldName": "aws.cloudwatch.dimensions.Service",
+                "id": "1549397251041",
+                "indexPatternRefName": "control_0_index_pattern",
+                "label": "AWS Services",
```
Comment (mtojek) [inline x-pack/metricbeat/module/aws/_meta/kibana/7/dashboard/Metricbeat-aws-usage-overview.json]: nit: Shouldn't it be `AWS services` to be consistent?

Comment (kaiyan-sheng) [reply x-pack/metricbeat/module/aws/_meta/kibana/7/dashboard/Metricbeat-aws-usage-overview.json]: Good point! I will change it in the dashboard!

Comment (kaiyan-sheng) [reply x-pack/metricbeat/module/aws/_meta/kibana/7/dashboard/Metricbeat-aws-usage-overview.json]: done.

**Comment by mtojek** [comment]
Manual testing passed.

I see usage data collected periodically, dashboard imported without any issues, dashboard presents real time data, filters works.

Well done!

**Pull request merged**